File: apps/docs/src/code/python/cwd/process.py
from e2b import Sandbox

sandbox = Sandbox(template="base", cwd="/code")  # $HighlightLine
sandbox_cwd = sandbox.process.start("pwd")  # $HighlightLine
sandbox_cwd.wait()
print(sandbox_cwd.output.stdout)
# output: "/code"

process_cwd = sandbox.process.start("pwd", cwd="/home")  # $HighlightLine
process_cwd.wait()
print(process_cwd.output.stdout)
# output: "/home"

sandbox.close()

File: apps/docs/src/code/js/cwd/process.js
import { Sandbox } from '@e2b/sdk'

const sandbox = await Sandbox.create({
  id: 'base',
  cwd: '/code', // $HighlightLine
})

const sandboxCwd = await sandbox.process.start({cmd: 'pwd'}) // $HighlightLine
await sandboxCwd.wait()
console.log(sandboxCwd.output.stdout)
// output: /code

const processCwd = await sandbox.process.start({cmd: 'pwd', cwd: '/home'}) // $HighlightLine
await processCwd.wait()
console.log(processCwd.output.stdout)
// output: /home

await sandbox.close()
